### PR TITLE
Add title props for image component

### LINKED ISSUE #2367: ✨ Component Image / Add a new prop "title"
### Is your iteration request related to a problem?

It's not possible to set a title in the current component `image`

### Describe the solution you'd like

A new `prop` for metadata "title"

### Screenshots

_No response_

### Additional context**

_No response_

### Additional info

Initially reported by @Osgar in the audit for Fotocasa, thnx!

### PR DESCRIPTION
**Pull request by Ph0tonic**
## Atom/Image

**TASK**: https://github.com/SUI-Components/sui-components/issues/2367

### Types of changes
Add props `title` to `image` component

- [ ] 🪲 Bug fix (non-breaking change which fixes an issue)
- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 🧾 Documentation
- [ ] 📷 Demo
- [ ] 🧪 Test
- [ ] 🧠 Refactor
- [ ] 💄 Styles

### CODE REVIEW

**Comment by turolopezsanabria** [comment]
thnx @Ph0tonic !

@Osgar and @nucliweb can you check this PR ir it has everything you need please? 
I think the initial request is related to SEO needs, that we don't leave anything out of images.

thnx!

File: components/atom/image/src/types.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ const htmlImgProps = {
   sizes: PropTypes.string,
   srcset: PropTypes.string,
   usemap: PropTypes.string,
-  width: PropTypes.string
+  width: PropTypes.string,
+  title: PropTypes.string
```
Comment (nucliweb) [inline components/atom/image/src/types.js]: Can you change to alphabetical order?

```
  sizes: PropTypes.string,
  srcset: PropTypes.string,
  title: PropTypes.string
  usemap: PropTypes.string,
  width: PropTypes.string,
```

Comment (Ph0tonic) [reply components/atom/image/src/types.js]: Done 👍

**Review by nucliweb — APPROVED** [review]
Thanks @Ph0tonic 😊

**Review by andresin87 — APPROVED** [review]

**Pull request merged**